### DISCUSSION TITLE
How to get JWT token

### DISCUSSION THREAD

**Discussion by pilapilabear** [lf-edge/ekuiper#3337] [Q&A]
File: [/docs/zh_CN/api/restapi/authentication.md](https://ekuiper.org/docs/zh/latest/api/restapi/authentication.html)

**Comment by ngjaying** [comment]
It is a RSA signed JWT token. Check https://github.com/lf-edge/ekuiper/blob/master/internal/pkg/jwt/jwt_rsa.go  CreateToken function for detail.
